usr/bin/ublue-user-setup

```diff
@@ -17,6 +17,13 @@ IMAGE_NAME=$(jq -r '."image-name"' < $IMAGE_INFO)
 # Enable NTP
 timedatectl set-ntp true
 
+if [[ $IMAGE_NAME = "bluefin-dx" ]]; then
+
+  source /usr/bin/bluefin-dx-user-setup
```

Comment:
If you keep this hook, increment the script version, `USER_SETUP_VER`, so that existing users get the new stuff